Cargo.toml

```diff
@@ -41,7 +41,7 @@ owning_ref = "0.4"
 stable_deref_trait = "1.0.0"
 rust-stemmers = "1"
 downcast-rs = { version="1.0" }
-matches = "0.1"
+matches = "0.1.8"
```

Comment:
Was the fix in `matches` or in `rustc` ? If `rustc` let's not fix the minor.

Reply:
ok with fixing the patch here.

Reply:
Will do